code/modules/clothing/masks/gasmask.dm

```diff
@@ -320,4 +320,92 @@
 		new /mob/living/simple_animal/hostile/retaliate/cluwne/psychedelicgoblin(get_turf(src))
 		qdel(W)
-		qdel(src)
\ No newline at end of file
+		qdel(src)
+
+/obj/item/clothing/mask/gas/hecu
+	name = "HECU gas mask"
+	desc = "An ancient gas mask with the letters HECU stamped on the side. Comes with a built-in voice modulator that slowly recharges."
+	icon_state = "hecu"
+	can_flip = 0
+	canstage = 0
+	ignore_flip = 1
+	flags = HEAR | FPRINT
+	var/togglestate = 1
+	var/max_charge = 100
+	var/mask_charge = 100
+	var/word_cost = 7
+	var/word_delay = 7
+	actions_types = list(/datum/action/item_action/toggle_voice)
+
+	//Big list of words pulled from half life's soldiers, used for both matching with spoken text and part of the sound file's path
+	var/list/hecuwords = list(
+		"a", "affirmative", "alert", "alien", "all" , "am" , "anything" , "are" , "area" , "ass" , "at" , "away" ,
+		"backup" , "bag" , "bastard" , "blow" , "bogies" , "bravo" , "call" , "casualties" , "charlie" , "check" , "checking" , "clear" , "comma" ,
+		"command" , "continue" , "control" , "cover" , "creeps" , "damn" , "delta" , "down" , "east" , "echo" , "eliminate" , "everything" , "fall" ,
+		"fight" , "fire" , "five" , "force" , "formation" , "four" , "foxtrot" , "freeman" , "get" , "go" , "god" , "going" , "got" , "grenade" , "guard" ,
+		"haha" , "have" , "he" , "heavy" , "hell" , "here" , "hold" , "hole" , "hostiles" , "hot" , "i" , "in" , "is" , "kick" , "killcivvies" ,
+		"killscientists" , "lay" , "left" , "lets" , "level" , "lookout" , "maintain" , "mission" , "mister" , "mother" , "move" , "movement" , "moves" ,
+		"my" , "need" , "negative" , "neutralize" , "neutralized" , "nine" , "no" , "north" , "nothing" , "objective" , "of" , "oh" , "okay" , "one" ,
+		"orders" , "our" , "out" , "over" , "patrol" , "people" , "period" , "position" , "post" , "private" , "quiet" , "radio" , "recon" , "request" ,
+		"right" , "roger" , "sector" , "secure" , "shit" , "shot" , "sign" , "signs" , "silence" , "sir" , "six" , "some" , "something" , "south" , "squad" ,
+		"stay" , "suppressing" , "sweep" , "take" , "tango" , "target" , "team" , "that" , "thatbastard" , "the" , "there" , "these" , "this" , "those" ,
+		"three" , "tight" , "two" , "uh" , "under" , "up" , "we" , "weapons" , "weird" , "west" , "we've" , "whatbody" , "whoisfreeman" , "will" , "yeah" ,
+		"yes" , "yessir" , "you" , "your" , "zero" , "zone" , "zulu" , "meters" , "seven" , "eight" , "hundred" , "to" , "too"
+		)
+
+//Toggling the mask so the user can let it recharge and not be annoying.
+/obj/item/clothing/mask/gas/hecu/verb/toggle_voice()
+	set src in usr
+	set name = "Toggle Voice Modulator"
+	set category = "Object"
+	if (!usr || loc != usr)
+		return
+	return togglevoice(usr)
+
+/obj/item/clothing/mask/gas/hecu/attack_self()
+	src.togglevoice()
+	..()
+	return
+
+/obj/item/clothing/mask/gas/hecu/proc/togglevoice(var/mob/user = usr)
+	if(usr.isUnconscious())
```

Comment:
Replace `usr` by `user`. `usr` is a byond internal variable which we prefer not to use.